target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -134,8 +134,9 @@ huasifei,ws1208v2)
 	ucidef_set_led_netdev "wwan0" "wwan0" "green:cellular" "wwan0" "link tx rx"
 	;;
 keenetic,kn-1910|\
-keenetic,kn-3010)
-	ucidef_set_led_netdev "internet" "internet" "green:internet" "wan"
+keenetic,kn-3010|\
```

Comment:
The KN-3010 uses green:internet for the LED. This change breaks already supported devices.